test/ODataEndToEnd/Microsoft.OData.Service.Sample.Northwind.Tests/Baselines/TestGetNorthwindMetadata.txt

```diff
@@ -12,34 +12,71 @@
         <Property Name="Picture" Type="Edm.Binary" />
         <NavigationProperty Name="Products" Type="Collection(Microsoft.OData.Service.Sample.Northwind.Models.Product)" />
       </EntityType>
-      <EntityType Name="Contact">
+      <EntityType Name="Product">
```

Comment:
Already have detail compare, the changes in this file is only sequence change, no single line content change.